src/static/js/pluginfw/shared.js

```diff
@@ -50,20 +51,23 @@ function extractHooks(parts, hook_set_name, normalizer) {
 
           const disabledReason = (disabledHookReasons[hook_set_name] || {})[hook_name];
           if (disabledReason) {
-            console.error(`Hook ${hook_set_name}/${hook_name} is disabled. Reason: ${disabledReason}`);
+            console.error(`Hook ${hook_set_name}/${hook_name} is disabled.
+                Reason: ${disabledReason}`);
             console.error(`The hook function ${hook_fn_name} from plugin ${part.plugin} ` +
                       'will never be called, which may cause the plugin to fail');
-            console.error(`Please update the ${part.plugin} plugin to not use the ${hook_name} hook`);
+            console.error(`Please update the ${part.plugin} plugin to not
+                use the ${hook_name} hook`);
             return;
           }
-
+          let hook_fn;
           try {
-            var hook_fn = loadFn(hook_fn_name, hook_name);
+            hook_fn = loadFn(hook_fn_name, hook_name);
             if (!hook_fn) {
-              throw 'Not a function';
+              throw new Error('Not a function');
             }
           } catch (exc) {
-            console.error(`Failed to load '${hook_fn_name}' for '${part.full_name}/${hook_set_name}/${hook_name}': ${exc.toString()}`);
+            console.error(`Failed to load '${hook_fn_name}' for
+              '${part.full_name}/${hook_set_name}/${hook_name}': ${exc.toString()}`);
```

Comment:
Don't introduce newlines.

```suggestion
            console.error(`Failed to load '${hook_fn_name}' for ` +
                          `'${part.full_name}/${hook_set_name}/${hook_name}': ${exc.toString()}`);
```